### PR TITLE
[Feature five] New feature 5 was implemented

### PR DESCRIPTION
**Pull request by alemafg**
i have created new folder featureFive with 2 files

### CODE REVIEW

File: feature5/another-file5.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Some other dumy text
```
Comment (vkrishna11) [inline feature5/another-file5.txt:1]: This is dummy review

Comment (martiserra46) [inline feature5/another-file5.txt:1]: It is really nice that you added this line of code

Comment (Github22799) [inline feature5/another-file5.txt:1]: abcd

File: feature5/new-file5.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+New file in the new-file5 branch
```
Comment (Github22799) [inline feature5/new-file5.txt:1]: aaaaaaaaaa

**Pull request closed without merging**